site/docs/voice/about.mdx

```diff
@@ -49,7 +57,37 @@ When calls or call events occur - Bandwidth will send a request (webhook) to you
 
 ### Webhooks
 
-The Voice API uses a number of different [webhooks](/docs/voice/webhooks) to facilitate call control. Certian webhooks expect BXML in the HTTP response to keep the call alive, while others are simply imformative.
+The Voice API uses a number of different [webhooks](/docs/voice/webhooks) to facilitate call control. Certain webhooks expect BXML in the HTTP response to keep the call alive, while others are simply informative. These used to be referred to as "callbacks".
```

Comment:
Looks like we still have about 82 references to "callback" still in the voice docs, either we should clean those up, or change this to say `These may also be referred to as callbacks`

```
grep -r callback ./api-docs/site/docs/voice | wc -l
      82
```

Reply:
I think we just need a full story for cleaning up these discrepancies in the docs. Referring to things by 2 names in documentation seems bad, but I don't know if it fits in this story. Should I add it to the cleanup PR?

Reply:
We use callback in our API webhooks and verbs, so this is being left for now.